core/src/main/java/org/mskcc/cbio/portal/scripts/ImportSampleList.java

```diff
@@ -132,6 +132,11 @@ public static void importSampleList(File dataFile) throws IOException, DaoExcept
 
       sampleList = daoSampleList.getSampleListByStableId(stableId);
 
+      if (stableId.contains("_sequenced")) {
+         // Temporary create new format of SV data from fusion data
+         DaoMutation.createStructuralVariantData(cancerStudyId);
+      }
+
```

Comment:
Not sure if the check should be in import sample list. I think it should be after import of mutation data. I'm not sure if we can assume the sample list gets imported after the mutation data

Reply:
agree with Ino, we should not make an assumption on the order in which the data got inserted.
Also, this would not work for studies that have the structural variants case list included. For example msk-impact-2017 study. 
in general, we should support two use cases:
1. importing a study that has a _sv case list 
2. importing a study without a _sv case list supplied.